### PR TITLE
Verify gem size during deployment and reduce fastlane size

### PR DESCRIPTION
**Pull request by KrauseFx**
screenshot 2017-12-18 08 38 25

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -301,6 +308,10 @@ private_lane :validate_repo do
     sh "bundle check || bundle install"
     sh "rake install"
 
+    # Ensure the file size of the fastlane gem is below 1 MB
+    size_in_mb = File.size("pkg/fastlane-#{version}.gem").to_f / 1_024_000
```
Comment (giginet) [inline fastlane/Fastfile]: Actually, you should divide by `(1024 * 1024)` ?

**Review by giginet — APPROVED** [review]
🚀

File: fastlane.gemspec [hunk 2]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.0.0'
 
-  spec.files = Dir.glob("*/lib/**/*", File::FNM_DOTMATCH) + Dir["fastlane/swift/**/*"] + Dir["bin/*"] + Dir["*/README.md"] + %w(README.md LICENSE) - Dir["fastlane/lib/fastlane/actions/device_grid/assets/*"]
+  spec.files = Dir.glob("*/lib/**/*", File::FNM_DOTMATCH) + Dir["fastlane/swift/**/*"] + Dir["bin/*"] + Dir["*/README.md"] + %w(README.md LICENSE) - Dir["fastlane/lib/fastlane/actions/device_grid/assets/*"] - Dir["fastlane/lib/fastlane/actions/docs/assets/*"]
```
Comment (giginet) [inline fastlane.gemspec]: Should you define white_lists?

Comment (KrauseFx) [reply fastlane.gemspec]: Yeah great idea 👍

**Pull request merged**